### PR TITLE
NTR-astrocyte-restricted-precursor-and-neuronal-restricted-precursor

### LINKED ISSUE #2435: [NTR] Astrocyte-restricted precursor
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl
--> not available

**Preferred term label**
astrocyte-restricted precursor

**Synonyms** (add reference(s), please)
ARP

https://singlecell.broadinstitute.org/single_cell/study/SCP263/aging-mouse-brain#study-summary

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
tentative definition: A neural precursor cell that undergoes a gliogenic switch to generate astrocytes.
https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2023.1063843/full

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
is_a: CL:0000123 ! neuron associated cell (sensu Vertebrata)
is_a: CL:0000030 ! glioblast ??
is_a: CL:0011026 ! progenitor cell

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
UBERON:0000955 ! brain

**Your ORCID**
https://orcid.org/0000-0003-3308-6245

**Additional notes or concerns**
It exists CL:4040000 glial restricted tripotential precursor cell but the definition is restricted to oligodendrocytes and type-1 and type-2 astrocytes, while there are 3 astrocyte sub-types, see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122459/

### LINKED ISSUE #2437: [NTR] neuronal-restricted precursor
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl
--> not available

**Preferred term label**
neuronal-restricted precursor

**Synonyms** (add reference(s), please)
NRP

https://singlecell.broadinstitute.org/single_cell/study/SCP263/aging-mouse-brain#study-summary

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
https://link.springer.com/chapter/10.1007/978-1-59259-107-7_5
'populations of restricted progenitors isolated from the embryonic spinal cord and the neonatal forebrain anterior subventricular
zone (SVZa), which are committed to the production of neurons and are therefore called neuronal restricted progenitors (NRPs).

but also
https://www.frontiersin.org/journals/neuroanatomy/articles/10.3389/fnana.2018.00104/full
--> 'More recently, it has been shown that the adult human hypothalamus contains four distinct populations of cells that express neuronal **progenitor** markers, each of these cell types, with the exception of tanycytes, are human-specific ([Pellegrino et al., 2018](https://www.frontiersin.org/journals/neuroanatomy/articles/10.3389/fnana.2018.00104/full#B64)).

Tricky to define 'neuronal-restricted precursor', in the reference cited above (Pellegrino et al., 2018), we see:
'neural stem/progenitor cell (NPC)'

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
is_a: CL:0011026 ! progenitor cell

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
UBERON:0000955 ! brain

**Your ORCID**
https://orcid.org/0000-0003-3308-6245

**Additional notes or concerns**
This issue is related to the 'Atlas of the Aging Mouse Brain'
https://singlecell.broadinstitute.org/single_cell/study/SCP263/aging-mouse-brain#study-summary
that reports cell types, some of them not (yet?) in CL/UBERON

### PR DESCRIPTION
**Pull request by AvolaAmg**
fixes #2437 fixes #2435

### CODE REVIEW

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by AvolaAmg** [comment]
#gogoeditdiff

**Review by Caroline-99 — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32321,6 +32323,33 @@ SubClassOf(obo:CL_4042020 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000625
 SubClassOf(obo:CL_4042020 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000007479))
 SubClassOf(obo:CL_4042020 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000007489))
 
+# Class: obo:CL_4042021 (neuronal-restricted precursor)
+
```
[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl]: unsure on the neuroepithelial development so I would not mention it

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
everything looks good

**Pull request merged**